lammps/src/COLVARS/colvarproxy_lammps.cpp

```diff
@@ -153,8 +150,12 @@ colvarproxy_lammps::~colvarproxy_lammps()
 // re-initialize data where needed
 int colvarproxy_lammps::setup()
 {
+  int error_code = colvarproxy::setup();
```

Comment:
Thanks, I forgot to add it to VMD and GROMACS (maybe precisely because it's empty and so it's inconsequential now). But having the unspecialized function there is the first step toward condensing code, as in previous tidyups of the proxy class.